src/locales/vocabulary.js

```diff
@@ -54,23 +54,23 @@ export default {
   'featurepanel.error_network': "Can't get the feature, check your network cable.",
   'featurepanel.error_deleted': 'This object is marked as deleted in OpenStreetMap.',
   'featurepanel.history_button': 'History »',
-  'featurepanel.other_info_heading': 'More information',
+  'featurepanel.other_info_heading': 'Details',
   'featurepanel.edit_button_title': 'Edit in OpenStreetMap database',
-  'featurepanel.edit_button': 'Edit place',
-  'featurepanel.add_place_button': 'Add place',
+  'featurepanel.edit_button': 'Suggest an edit',
```

Comment:
It is used for actual edit as well. 

See https://github.com/zbycz/osmapp/issues/96#issuecomment-1398740261

Reply:
Added a ``note_button`` entry for users not logged in.

Reply:
Looks good. Thanks. If i pushed another commit with the code in this PR, would you manage to add other edits? I think its ok for web editation. But it can be tricky on local machine.

Reply:
@zbycz not sure what you mean by this, sorry, I'm new-ish to pull requests

Reply:
Don't worry. I will do it differently. 🙂